profile/README.md

```diff
@@ -0,0 +1,63 @@
+# Lombiq Technologies 
```

Comment:
I think your branch name should've been `issue/BIZ-43`? Nothing to do about it now. Please include the issue code in the pull request title (so I guess "BIZ-44: Github organization readme file") and in the description too.